### PR TITLE
wrap retro envs correctly for other (non-deepq) algorithms

### PR DESCRIPTION
**Pull request by pzhokhov**
this enables, for instance, ppo2 on retro games (related to this: https://github.com/openai/baselines/issues/646)

### CODE REVIEW

File: baselines/common/cmd_util.py [hunk 1]
```diff
@@ -16,30 +16,58 @@
...
     set_global_seeds(seed)
-    if num_env > 1: return SubprocVecEnv([make_env(i + start_index) for i in range(num_env)])
-    else: return DummyVecEnv([make_env(start_index)])
+    if num_env > 1:
+        return SubprocVecEnv([make_thunk(i + start_index) for i in range(num_env)])
+    else:
+        return DummyVecEnv([make_thunk(start_index)])
+
+
+def env_thunk(env_id, env_type, subrank=0, seed=None, reward_scale=1.0, gamestate=None, wrapper_kwargs=None):
```
Comment (christopherhesse) [inline baselines/common/cmd_util.py]: should this be make_env to correspond to make_vec_env above?

File: baselines/common/cmd_util.py [hunk 2]
```diff
@@ -16,30 +16,58 @@
 from baselines.common.atari_wrappers import make_atari, wrap_deepmind
 from baselines.common.vec_env.subproc_vec_env import SubprocVecEnv
 from baselines.common.vec_env.dummy_vec_env import DummyVecEnv
-from baselines.common.retro_wrappers import RewardScaler
+from baselines.common import retro_wrappers
+import functools
```
Comment (christopherhesse) [inline baselines/common/cmd_util.py]: I think this isn't used anywhere

Comment (pzhokhov) [reply baselines/common/cmd_util.py]: yes, thanks! flake8 flags it now (was delayed due to github issues)

**Comment by Omegastick** [comment]
I think this broke `make_vec_env` for Atari environments. `wrapper_kwargs` isn't passed in `make_thunk` so it is always `None`. This gives `TypeError: wrap_deepmind() argument after ** must be a mapping, not NoneType` when you call it. I'll open an issue.

**Pull request merged**